src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3306,6 +3318,64 @@ private void AddAccessorIfAvailable(ArrayBuilder<Symbol> symbols, MethodSymbol a
             }
         }
 
+        internal override byte? GetLocalNullableContextValue()
+        {
+            byte? value;
+            if (!_flags.TryGetNullableContext(out value))
+            {
+                value = ComputeNullableContextValue();
+                _flags.SetNullableContext(value);
+            }
+            return value;
+        }
+
+        private byte? ComputeNullableContextValue()
+        {
+            var compilation = DeclaringCompilation;
+            if (!compilation.ShouldEmitNullableAttributes(this))
+            {
+                return null;
+            }
+
+            var builder = new MostCommonNullableValueBuilder();
+            var baseType = BaseTypeNoUseSiteDiagnostics;
+            if (!(baseType is null))
+            {
+                builder.AddValue(TypeWithAnnotations.Create(baseType));
+            }
+            foreach (var typeParameter in TypeParameters)
+            {
+                typeParameter.GetCommonNullableValues(compilation, ref builder);
+            }
+            foreach (var pair in GetMembersByName())
```

Comment:
>GetMembersByName [](start = 33, length = 16)

Is the order significant? Can we simply use GetTypeMembersUnordered?

Reply:
`GetTypeMembersUnordered` contains the nested types only.

---
In reply to: [296902624](https://github.com/dotnet/roslyn/pull/36152#discussion_r296902624) [](ancestors = 296902624)

Reply:
> GetTypeMembersUnordered contains the nested types only.

I meant ```GetMembersUnordered```.

---
In reply to: [296904137](https://github.com/dotnet/roslyn/pull/36152#discussion_r296904137) [](ancestors = 296904137,296902624)